### PR TITLE
[SPARK-33367][SQL] Move the table commands from ddl.scala to table.scala

### PR DESCRIPTION
**Pull request by imback82**
### What changes were proposed in this pull request?

This PR addresses the following TODO and moves the table commands from `ddl.scala` to `table.scala`:
https://github.com/apache/spark/blob/4941b7ae18d4081233953cc11328645d0b4cf208/sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala#L137

### Why are the changes needed?

To have table-related commands in the same file.

### Does this PR introduce _any_ user-facing change?

No, this is just refactoring.

### How was this patch tested?

Existing tests.

### CODE REVIEW

File: sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala [hunk 1]
```diff
@@ -133,9 +141,6 @@ case class CreateTableLikeCommand(
   }
 }
 
-
-// TODO: move the rest of the table commands from ddl.scala to this file
```
Comment (imback82) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala:137]: This PR addresses this TODO.

**Comment by HyukjinKwon** [comment]
Hey, I don't think this is minor. Let's create new JIRA.

**Comment by imback82** [comment]
retest this please

**Review by dongjoon-hyun — COMMENTED** [review]
I believe @gatorsmile may want to review this because of the loss of the commit history.

cc @HeartSaVioR

**Comment by imback82** [comment]
Thanks @dongjoon-hyun.

Please let me know if this refactoring is not desirable due to commit history, etc., then I can just close this PR.

**Comment by gatorsmile** [comment]
Yes. I would keep it unchanged if possible. Just easy to check the change history for new contributors.

**Comment by imback82** [comment]
OK, I will close this then. Thanks!

**Comment by HeartSaVioR** [comment]
I think you can change your PR to remove TODO. TODOs in codebase is quite confusing for contributors, so once we confirm we don't want to deal with that, we should also remove the TODO.

**Pull request closed without merging**